solutions/devsprint-firas-safa-1/GitHubApp/Screens/Settings/SettingsView.swift

```diff
@@ -7,6 +7,62 @@
 
 import UIKit
 
+private let cellIdentifier = "SettingsCell"
+
 class SettingsView: UIView {
+    private let viewModel: SettingsViewModel
+    private let tableView = UITableView()
```

Comment:
first the UI components, then the viewmodel
so switch the order of the variales and put a space between them